bitnami/contour/templates/contour/ingressclass.yaml

```
@@ -0,0 +1,16 @@
+{{ $ingressClass := .Values.contour.ingressClass }}
+{{- if (kindIs "map" $ingressClass) }}
+{{- if $ingressClass.create }}
+---
+apiVersion: networking.k8s.io/v1
+kind: IngressClass
+metadata:
+  name: {{ $ingressClass.name }}
+  annotations:
+    ingressclass.kubernetes.io/is-default-class: {{ $ingressClass.default | quote }}
```

We should include the namespace:
```suggestion
  namespace: {{ .Release.Namespace | quote }}
  annotations:
    ingressclass.kubernetes.io/is-default-class: {{ $ingressClass.default | quote }}
```

---

Also.. should we support custom annotations?

---

> We should include the namespace:

Since IngressClass is Cluster resource, we don't have to include namespace 🙂

> Also.. should we support custom annotations?

On configuration perspective, I think we don't have to support custom annotations 🤔. (As far as I know, `ingressclass.kubernetes.io/is-default-class` is the only valid option for now.)

Namespace scoped parameters (`.spec. parameters`) might be a viable option, but I think it would be better to keep as-is because contour doesn't support it.

---

Fair enough!!